macOS/DuckDuckGo/NavigationBar/View/AddressBarButtonsViewController.swift

```diff
@@ -399,6 +399,20 @@ final class AddressBarButtonsViewController: NSViewController {
 
     override func viewWillAppear() {
         setupButtons()
+
+        // Store reference to DraggingDestinationView for popup window dragging
+        if isInPopUpWindow {
+            guard let customView = view as? AddressBarButtonsView else {
+                assertionFailure("AddressBarButtonsViewController.view should be AddressBarButtonsView")
+                return
+            }
+            assert(type(of: view.superview) == NSView?.self)
+            guard let nextResponder = view.superview?.nextResponder as? DraggingDestinationView else {
+                assertionFailure("Expected DraggingDestinationView as next responder, got \(type(of: nextResponder))")
```

Comment:
I'm guessing you want to use `view.superview?.nextResponder` instead as `nextResponder` is `nil` here.

Reply:
right!